rdmd_test.d

```diff
@@ -684,11 +684,11 @@ void runFallbackTest(string rdmdApp, string buildCompiler, string model)
        if an explicit --compiler flag is not provided, rdmd should
        search its own binary path first when looking for the default
        compiler (determined by the compiler used to build it) */
-    string localDMD = buildPath(tempDir(), baseName(buildCompiler).setExtension(binExt));
+    string localDMD = buildPath(dirName(rdmdApp), baseName(buildCompiler).setExtension(binExt));
```

Comment:
No functional change, just clarifies things IMO (no need to know that `rdmdApp` lies in the `tempDir()` directory).